tests/test_audio.py

```diff
@@ -156,6 +168,24 @@ def test_load_timestamp_before_warnmode(metadata_wav_str):
         assert s.metadata["recording_start_time"] == correct_ts
 
 
+def test_retain_metadata(metadata_wav_str, new_metadata_wav_str):
+    a = Audio.from_file(metadata_wav_str)
+    a.save(new_metadata_wav_str)
+    new_a = Audio.from_file(new_metadata_wav_str)
+
+    # file size may differ slightly, other fields should be the same
+    new_a.metadata["filesize"] = None
+    a.metadata["filesize"] = None
+    assert new_a.metadata == a.metadata
+
+
+def test_update_metadata(metadata_wav_str, new_metadata_wav_str):
+    a = Audio.from_file(metadata_wav_str)
+    a.metadata["artist"] = "newartist"
```

Comment:
Same comment as for retain test, instead of testing one random field, could we not loop through all fields and verify.

Reply:
the point of this test is simply to check if a change to the metadata is retained across saving and loading a file

Reply:
Which should be verified for all fields in my opinion, can be done through a dict as well, to compare entire objects as done in retain test (which I had missed).